feature/profile/src/main/res/values/colors.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
We have Colors.kt in designsystem module move these their or check it is already present to not.